devicetypes/smartthings/zooz-multisiren.src/zooz-multisiren.groovy

```diff
@@ -0,0 +1,198 @@
+		 state "battery", label:'${currentValue}% battery', unit:""
+	 }
+	
+	standardTile("refresh", "command.refresh", inactiveLabel: false, decoration: "flat", width: 2, height: 2) {
+		state "default", label:'', action:"refresh.refresh", icon:"st.secondary.refresh"
+	}
+
+	valueTile("tamper", "device.tamper", height: 2, width: 2, decoration: "flat") {
+		state "clear", label: 'tamper clear', backgroundColor: "#ffffff"
+		state "detected", label: 'tampered', backgroundColor: "#ff0000"
+	}
+   
+	main "alarm"
+	details(["alarm", "humidity", "battery", "temperature", "tamper", "refresh"])
+	
+	}
+}
+
+def installed() {
+	initialize()
+}
+
+def refresh() {
+	def commands = []
+	 //get temperature value
+	commands << secure(zwave.sensorMultilevelV5.sensorMultilevelGet(sensorType: 0x01))
+	//get humidity value
+	commands << secure(zwave.sensorMultilevelV5.sensorMultilevelGet(sensorType: 0x05))
+	//get tamper state
+	commands << secure(zwave.notificationV3.notificationGet(notificationType: 0x07))
+	//get state of device (on or off)
+	commands << secure(zwave.basicV1.basicGet())
+	//get battery value
+	commands << secure(zwave.batteryV1.batteryGet())
+	
+	commands
+}
+
+def initialize() {
+	sendEvent(name: "checkInterval", value: 2 * 60 * 60 + 2 * 60, displayed: true, data: [protocol: "zwave", hubHardwareId: device.hub.hardwareID, offlinePingable: "1"])
+	def cmd = []
+	//temperature and humidity are set for reporting every 60 min
+	cmd << secure(zwave.configurationV1.configurationSet(parameterNumber: 2, size: 2, configurationValue: [60]))
+	return response(cmd << refresh())
+}
+
+def configure() {
+	initialize()
+}
+
+def parse(String description) {
+	def result = null
+
+	def cmd = zwave.parse(description)
+	if (cmd) {
+		result = zwaveEvent(cmd)
+	}
+	
+	return result
+}
+
+def zwaveEvent(physicalgraph.zwave.commands.basicv1.BasicReport cmd) {
+	createEvents(cmd)
+}
+
+def zwaveEvent(physicalgraph.zwave.commands.switchbinaryv1.SwitchBinaryReport cmd) {
+	createEvents(cmd)
+}
+
+private createEvents(cmd) {
+	[
+		createEvent([name: "switch", value: cmd.value ? "on" : "off"]),
+		createEvent([name: "alarm", value: cmd.value ? "both" : "off"])
+	]
+}
+
+def zwaveEvent(physicalgraph.zwave.commands.notificationv3.NotificationReport cmd) {
+	//when opening device cover
+	if(cmd.notificationType == 7) {
+		if(cmd.event == 3) {
+			sendEvent(name: "tamper", value: "detected")
+		} else {
+			sendEvent(name: "tamper", value: "clear")
+		}
+	}
+}
+
+def zwaveEvent(physicalgraph.zwave.commands.sensormultilevelv5.SensorMultilevelReport cmd) {
+	if(cmd.sensorType == 1) {
+		sendEvent(name: "temperature", value: cmd.scaledSensorValue)
+	} else if(cmd.sensorType == 5) {
+		sendEvent(name: "humidity", value: cmd.scaledSensorValue)
+	}
+}
+
+def zwaveEvent(physicalgraph.zwave.commands.batteryv1.BatteryReport cmd) {
+	sendEvent(name: "battery", value: cmd.batteryLevel)
```

Comment:
please add the standard logic for low battery (`0xFF`).

Reply:
done